packages/components/src/number-control/stories/index.tsx

```diff
@@ -21,8 +26,13 @@ export default {
 	},
 };
```

Comment:
Forgot to mention — this is missing some boilerplate:

```js
	parameters: {
		controls: { expanded: true },
		docs: { source: { state: 'open' } },
	},
```

Reply:
Good point, I've added it !